studio/components/ui/PageTelemetry.tsx

```diff
@@ -59,14 +61,16 @@ const PageTelemetry = ({ children }: PropsWithChildren<{}>) => {
         },
       })
 
-      post(`${API_URL}/telemetry/pageview`, {
-        ...(ref && { projectRef: ref }),
-        ...(selectedOrganization && { orgSlug: selectedOrganization.slug }),
-        referrer: referrer,
-        title: document.title,
-        path: router.route,
-        location: router.asPath,
-      })
+      if (isLoggedIn) {
```

Comment:
Just to confirm, we only want to register page views for logged in users? (aka we don't care about the views on the sign in pages?)

Reply:
@pamelachia - best to check with you about this. the only reason im guarding the pageview call is cause it currently 401s on the sign in pages

Reply:
oh if it's 401 already, I assume it's fine!